molr-commons/src/main/java/io/molr/commons/domain/Placeholder.java

```diff
@@ -33,6 +33,10 @@ private Placeholder(Class<T> type, String name) {
         return new Placeholder<>(String.class, name);
     }
 
+    public static final Placeholder<Void> aVoid(String name) {
+        return new Placeholder<>(Void.class, name);
+    }
+
```

Comment:
Mhmm .. do we really need this? (I did not think it through yet...) .. but having a placeholder for 'void' feels unnatural....

Reply:
probably we don't need this. If you look in other solutions like CompletableFuture, the Void type is just used to react on the end of an async task without results.. in our case the user could simply subscribe to the MissionState :smile:

Reply:
Yes, that makes sense. I will remove it.